malicious/npm/3a-look/MAL-0000-3a-look.json

```diff
@@ -0,0 +1,53 @@
+{
+  "modified": "2023-08-21T20:12:58Z",
+  "published": "2023-04-27T00:00:00Z",
+  "schema_version": "1.5.0",
+  "summary": "Malicious code in 3a-look (npm)",
+  "details": "Lazarus Group targeting blockchain and cryptocurrency companies by exploiting software supply chains through malicious npm packages and social engineering tactics",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "npm",
+        "name": "3a-look"
+      },
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0"
+            }
+          ]
+        }
+      ],
+      "database_specific": {
+        "cwes": [
+          {
+            "cweId": "CWE-506",
+            "description": "The product contains code that appears to be malicious in nature.",
+            "name": "Embedded Malicious Code"
+          }
+        ]
```

Comment:
I'm okay leaving this in - but a `cwes` field is not necessary.

The GHSA importer includes this, but it isn't generally required.

Reply:
got it. Just removed this.